tools/video_tools/pyscenedetect_shot_detection.xml

```diff
@@ -0,0 +1,20 @@
+<tool id="PySceneDetect_shot_detection" name="PySceneDetect" version="1.0.0">
+  <description>Runs shot detection on a video</description>
+    <requirements>
+    </requirements>
+  <command detect_errors="exit_code"> $__tool_directory__/pyscenedetect_shot_detection.py $input $threshold $amp_shots $frame_stats</command>
+  <inputs>
+	<param name="input" type="data" format="video" label="Video File" help="A video file input for shot detection"/>
+	<param name="threshold" type="integer" label="(Optional) Threshold" value="30" optional="true" help="Sensitivity of the shot detection. The default is 30"/>
```

Comment:
if you can set here a min and/or max like `min="0"` `max="100"` Galaxy will offer you a nice slider in the UI.